### ISSUE TITLE
New release?

### ISSUE THREAD

**Issue by Barough** [moisespr123/exhale-gui#2]
What about a new release that includes the latest exhale binary?

**Comment by moisespr123** [comment]
Yes, thanks for reminding me. It's been a long time since I used exhale. Will add this to my TODO list.

**Comment by dtseto** [comment]
This is good but it does not include latest binary that supports SBR which have different flags A-G.

**Comment by Barough** [comment]
@dtseto, Lastet Win binaries
[exhale-1.2.1.2-bffead0d_Win_GCC151.zip](https://github.com/user-attachments/files/21333599/exhale-1.2.1.2-bffead0d_Win_GCC151.zip)
